README.md

```diff
@@ -1,27 +1,27 @@
-# Core Sitemaps #
-**Contributors:** joemcgill, pacifika, kburgoine  
-**Tags:** seo, sitemaps  

```

Comment:
Please do not remove all these trailing spaces

Reply:
why?

Reply:
Because otherwise it looks like this:

![Screenshot 2020-01-28 at 15 38 52](https://user-images.githubusercontent.com/841956/73273477-4f7a8580-41e4-11ea-8b03-c6986bf7452b.png)

Reply:
I see. How about real line breaks instead of trailing spaces?

Reply:
No, that's how Markdown works.

By default, two spaces in markdown is the same as a `<br>` in HTML. Two line breaks would be `<p>`. We'd want `<br>` though.

I'll add a PR to update `.editorconfig` appropriately so this should be handled by your editor (if it supports the EditorConfig standard).

FWIW, your editor also seems to be changing all the line endings, causing changes to every line:

![Screenshot 2020-01-28 at 15 43 53](https://user-images.githubusercontent.com/841956/73273931-037c1080-41e5-11ea-8415-2012d2076485.png)

Are you on Windows by chance?

Reply:
Today I learned.

> Are you on Windows by chance?
I am on mac

Reply:
Actually, since the `readme.md` file is auto-generated using the built-in grunt task, you should not need to edit it manually. Running `npx grunt` should fix it.